### PR TITLE
[EWL-3573] : creates a partner logo card row

### PR DESCRIPTION
**Pull request by rpama**
## Ticket(s)

**Jira Ticket**

- [EWL-3573: EC | Design Partner Logo component](https://issues.ama-assn.org/browse/EWL-3573)

## Description

Created a row of linked images per wireframes

## To Test

- [x] Go to Organisms > EC > EC Partner Row
- [x] See that there are linked placeholder images in a row that cna hold up to 6 images across in desktop and that are 2 cols in width

## Relevant Screenshots/GIFs

None.

## Remaining Tasks

Could make molecules for the logo images themselves, but that seemed unnecessary given how simple they are.

## Additional Notes

Anything more to add?

### CODE REVIEW

File: styleguide/source/_patterns/02-organisms/05-ec/02-ec-partner-row/ec-partner-row.twig [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<div class="ec-partner-row layout">
```
Comment (deadarm) [inline styleguide/source/_patterns/02-organisms/05-ec/02-ec-partner-row/ec-partner-row.twig]: Suggestion to change the div to an unordered list element.

**Comment by rpama** [comment]
Updated to use existing logo atom for partner logos, converts div to list. Should be good to go, @deadarm. If you can take one more look...

**Comment by deadarm** [comment]
LGTM!

**Comment by deadarm** [comment]
Successfully merged.

**Pull request merged**